scripts/msct_image.py

```diff
@@ -1031,6 +1041,7 @@ def save_plane(self, plane='sagittal', index=None, format='.png', suffix='', seg
             index = [index]
 
         slice_list = [self.get_slice(plane=plane, index=i, seg=seg) for i in index]
+        path_output = sct.slash_at_the_end(path_output, 1)
```

Comment:
This `slash_at_the_end` looks like a remnant of the past. It shouldn't be necessary if `os.path.join()` id used.